docs/creating-new/cmake-dependencies.rst

```diff
@@ -70,8 +70,11 @@ In terms of CMake code it means adding ``HunterGate`` and ``hunter_add_package``
   :align: center
 
 Note that the SHA1 of the Hunter archive in ``HunterGate`` commands of ``Bar`` and
-``Roo`` can be different. Same location will be set automatically internally
-you don't have to manage it manually.
+``Roo`` can be different. The main project may contain subprojects also using the
+``HunterGate`` command. The subprojects may be introduced as hunter dependencies,
+git submodules, or just the whole project added as subfolder.
```

Comment:
you are right. Hunter uses something like SuperBuilds (done right :) )

Just for better understanding. Hunter dependencies are downloaded, extracted an build (ignoring the existence of cache servers). Only in the case of hunter dependencies, which are cmake-projects it is possible that the project has a `HunterGate` command.

The second case is a subproject added by `add_subdirectory` and this subproject uses the `HunterGate` command.

Do you know how we could name both cases with a single identifier?

And VCS (thus the mention of git submodules) should not be mentioned